src/Feliz.DaisyUI/DaisyUI.fs

```diff
@@ -207,9 +209,9 @@ type Daisy =
     static member inline link int = Helpers.Elm.int Html.a int "link"
     static member inline link float = Helpers.Elm.float Html.a float "link"
 
-//    static member inline mask props = Helpers.Elm.props Html.div props "mask"
-//    static member inline mask (children: #seq<ReactElement>) = Helpers.Elm.children Html.div children "mask"
-//    static member inline mask elm = Helpers.Elm.elm Html.div elm "mask"
+    static member inline mask props = Helpers.Elm.props Html.img props "mask"
+    static member inline mask (children: #seq<ReactElement>) = Helpers.Elm.children Html.img children "mask"
+    static member inline mask elm = Helpers.Elm.elm Html.img elm "mask"
```

Comment:
This one is not necessary, because mask can be applied to anything so I workarounded this over `mask` modifier which applies both classes

Reply:
Commented back out